test/ctmrg/gaugefix.jl

```diff
@@ -43,17 +45,17 @@ end
     psi = _make_symmetric(psi)
 
     Random.seed!(987654321)  # Seed RNG to make random environment consistent
+    # psi = InfinitePEPS(physical_space, peps_space; unitcell)
     ctm = CTMRGEnv(psi; Venv=ctm_space)
 
     verbosity = 1
     alg = CTMRG(;
-        tol=1e-10, miniter=4, maxiter=400, verbosity, trscheme=truncdim(dim(ctm_space))
+        tol=1e-10, miniter=4, maxiter=5000, verbosity, trscheme=FixedSpaceTruncation()
```

Comment:
This sounds like an excessive maxiter, do we need that?

Reply:
Yes definitely, that was just me trying around to stabilize the test. I will put it back to reasonable settings, once I stabilized it.